src/components/News/StaticChecklistContainer.jsx

```diff
@@ -27,7 +27,6 @@ const StaticChecklistContainer = () => {
             ml={-1}
             bgColor={boxColor}
             position="fixed"
-            boxShadow="dark-lg"
```

Comment:
I think we should keep this shadow since the checklist is sticky and sits above the rest of the page (in terms of zIndex), I think the shadow helps show that.

Look at the chat box on the bottom left here as an example of what I mean: https://bench.co/